src/app/channels/channel/channel.component.scss

```diff
@@ -0,0 +1,11 @@
+#header-image {
+    height: calc((100vw - 240px) / 6.2 - 1px);
```

Comment:
For better support in smaller screens, add `min-height`

Reply:
Done.